src/main/java/duke/command/GetTasks.java

```diff
@@ -0,0 +1,89 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidCommandException;
+import duke.exceptions.InvalidTaskNumberException;
+import duke.exceptions.EventTimingException;
+import duke.exceptions.DeadlineTimingException;
+
+import java.io.IOException;
+
+public class GetTasks extends DukeCommand {
+
+    public static void followCommand(String getUserInput) throws DukeException, IOException {
+        int taskNumber;
+        String userCommand = (getUserInput.split(" "))[0];
+        switch (userCommand) {
+        case "done":
+            description = getUserInput.split("done ");
+            if (description.length < 2 || Integer.parseInt(description[1]) == 0) {
+                throw new InvalidTaskNumberException();
+            }
+            taskNumber = Integer.parseInt(description[1]) - 1;
+            if (tasks.get(taskNumber) == null) {
+                throw new InvalidTaskNumberException();
+            }
```

Comment:
You can enclose this entire thing is one `try ... catch` block, find out how you can do that!